kubeconfig-exporter/kubernetes_export_sa.sh

```diff
@@ -2,23 +2,56 @@
 set -e
 set -o pipefail
 
-# Add user to k8s using service account, no RBAC (must create RBAC after this script)
-if [[ -z "$1" ]] || [[ -z "$2" ]] || [[ -z "$2" ]]; then
- echo "usage: $0 <service_account_name> <namespace>"
- echo "ex: sh ./kubeconfig-exporter/kubernetes_export_sa.sh cd-user cd-user"
- exit 1
+if [[ $# -lt 2 ]]; then
+  echo "usage: $0 <service_account_name> <namespace> [--devtron-endpoint=<value>] [--devtron-api-token=<value>] [--cluster-name=<value>] [--insecure=<value>] [--server_url=<value>]"
+  echo "ex: sh ./kubeconfig-exporter/kubernetes_export_sa.sh cd-user cd-user --cluster-name=cluster-name"
```

Comment:
keep the example same - `echo "ex: sh ./kubeconfig-exporter/kubernetes_export_sa.sh cd-user devtroncd"`